runtime/src/snapshot_utils.rs

```diff
@@ -2980,7 +2980,7 @@ mod tests {
         bank4.squash();
         bank4.clean_accounts(true, false, Some(full_snapshot_slot));
         assert_eq!(
-            bank4.get_account(&key1.pubkey()),
+            bank4.get_account_modified_slot(&key1.pubkey()),
```

Comment:
I think we can simplify to `assert!(bank4.get_account_modified_slot(&key1.pubkey()).is_none())`

Reply:
Ah yes, good catch!

Reply:
Done!